hoomd/md/alchemy/pair.py

```diff
@@ -364,3 +365,116 @@ def __init__(self,
                  mode='none'):
         _AlchemicalPairForce.__init__(self)
         super().__init__(nlist, default_r_cut, default_r_on, mode)
+
+
+@_modify_pair_cls_to_alchemical
+class OPP(BASEOPP, _AlchemicalPairForce):
```

Comment:
```suggestion
class OPP(BaseOPP, _AlchemicalPairForce):
```

Reply:
Better yet:
```class OPP(hoomd.md.pair.OPP, ...)```
Explicit is better than implicit.

Reply:
So using `BaseOPP` violates pep8-naming rule N811 because it thinks `OPP` is the name of a constant: https://peps.python.org/pep-0008/#constants

I think the simplest solution here is to be explicit with `class OPP(hoomd.md.pair.OPP, ...)` and then the imported alias is not necessary.

Reply:
Yeah that's why I replaced the name by Opp. I'll make it explicit. Thanks!